TimeService/scripts/leon-run.sh

```diff
@@ -0,0 +1,5 @@
+n7s-sis -gdb -port 1234 -leon3 -uart1 stdio ../work/binaries/partition_1 &
```

Comment:
This, if is to be included in the repo (which is a good idea), requires a clean-up